protoutil/blockutils_test.go

```diff
@@ -36,15 +38,17 @@ func TestNewBlock(t *testing.T) {
 	require.Equal(t, []byte("datahash"), block.Header.PreviousHash, "Incorrect previous hash")
 	require.NotNil(t, block.GetData())
 	require.NotNil(t, block.GetMetadata())
-	block.GetHeader().DataHash = protoutil.BlockDataHash(data)
+	block.GetHeader().DataHash, _ = protoutil.BlockDataHash(data)
+
+	dataHash, _ := protoutil.BlockDataHash(data)
```

Comment:
check no error?
Is there a test case that returns error?
Check when `data` is nil?